src/schema/v2/partner.ts

```diff
@@ -212,16 +241,43 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
         type: GraphQLBoolean,
         resolve: ({ pre_qualify }) => pre_qualify,
       },
-      locations: {
-        type: new GraphQLList(Location.type),
-        args: {
-          size: {
-            type: GraphQLInt,
-            defaultValue: 25,
-          },
+      locationsConnection: {
+        description: "A connection of locations from a Partner.",
+        type: locationsConnection.connectionType,
+        args: pageable({}),
+        resolve: ({ id }, args, { partnerLocationsLoader }) => {
+          const { page, size, offset } = convertConnectionArgsToGravityArgs(
+            args
+          )
+
+          interface GravityArgs {
+            exclude_ids?: string[]
+            page: number
+            published: boolean
+            size: number
+            total_count: boolean
+          }
```

Comment:
It’s great that you define types for these! But perhaps define them on the `partnerLocationsLoader` instead?

Reply:
```ts
partnerLocationsLoader: gravityLoader<
  { ... }, // response type, if helpful
  { exclude_ids?: string[], … }, // params type
>(id => `partner/${id}/locations`, {}, { headers: true }),
```

Reply:
Don't the params then need to be spread in the url? This isn't the convention we've been using. I'll take a look into it.

Reply:
OK moved these types!